applications/ConstitutiveLawsApplication/custom_constitutive/composites/rule_of_mixtures_law.h

```diff
@@ -477,6 +477,21 @@ class KRATOS_API(CONSTITUTIVE_LAWS_APPLICATION) ParallelRuleOfMixturesLaw
         const Vector& rShapeFunctionsValues
         ) override;
 
+    std::vector<ConstitutiveLaw::Pointer> &GetConstitutiveLaws()
+    {
+        return mConstitutiveLaws;
+    }
+
+    std::vector<double> &GetCombinationFactors()
```

Comment:
```suggestion
    std::vector<ConstitutiveLaw::Pointer>& GetConstitutiveLaws()
    {
        return mConstitutiveLaws;
    }

    std::vector<double>& GetCombinationFactors()
```
I prefer in the other side

Reply:
Done